apps/betterangels-backend/shelters/admin.py

```diff
@@ -2,118 +2,305 @@
 )
-from .models import (
-    SPA,
-    Accessibility,
-    CareerService,
-    City,
-    EntryRequirement,
-    Funder,
-    GeneralService,
-    HealthService,
-    ImmediateNeed,
-    Parking,
-    Pet,
-    Population,
-    Shelter,
-    ShelterType,
-    SleepingOption,
-    Storage,
-)
+from .models import ContactInfo, ExteriorPhoto, InteriorPhoto, Shelter, Video
 
 T = TypeVar("T", bound=models.Model)
+User = get_user_model()
 
 
 class ShelterForm(forms.ModelForm):
+    template_name = "admin/shelters/change_form.html"  # Specify your custom template path
+
+    # Summary Info
+    demographics = forms.MultipleChoiceField(
+        choices=DemographicChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select demographics...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=True,
+    )
+    demographics_other = forms.CharField(
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": "Please specify..."}),
+    )
+    special_situation_restrictions = forms.MultipleChoiceField(
+        choices=SpecialSituationRestrictionChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select special situation restrictions...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=True,
+    )
+    shelter_types = forms.MultipleChoiceField(
+        choices=ShelterChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select shelter types...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=False,
+    )
+    shelter_types_other = forms.CharField(
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": "Please specify..."}),
+    )
+
+    # Sleeping Details
+    room_styles = forms.MultipleChoiceField(
+        choices=RoomStyleChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select room style...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=False,
+    )
+    room_styles_other = forms.CharField(
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": "Please specify..."}),
+    )
+
+    # Shelter Details
+    accessibility = forms.MultipleChoiceField(
+        choices=AccessibilityChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select accessibility...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=False,
+    )
+    storage = forms.MultipleChoiceField(
+        choices=StorageChoices,
+        widget=Select2MultipleWidget(
+            attrs={
+                "data-placeholder": "Select storage...",
+                "data-allow-clear": "true",
+            }
+        ),
+        required=True,
```

Comment:
`storage`, `pets`, and `parking` aren't listed as required in the [source doc](https://docs.google.com/document/d/1dhPkGavRon-D2Ae4XJv1NUeYowJ3vl86npmI92ad4SU/edit?tab=t.0#heading=h.8amdd3r3eahw)